src/services/Serialization.jl

```diff
@@ -118,6 +118,17 @@ function packFactor(dfg::G, f::DFGFactor)::Dict{String, Any} where G <: Abstract
     return props
 end
 
+
+function decodePackedType(::Type{T}, packeddata::GenericFunctionNodeData{PT,<:AbstractString}) where {T<:FactorOperationalMemory, PT}
+  # usrtyp = convert(FunctorInferenceType, packeddata.fnc)
+  # Also look at parentmodule
+  usrtyp = getfield(PT.name.module, Symbol(string(PT.name.name)[7:end]))
+  fulltype = DFG.FunctionNodeData{T{usrtyp}}
+  factordata = convert(fulltype, packeddata)
+  return factordata
+end
```

Comment:
Moved here from IIF and signature changed to take the unpacking FactorOperationalMemory type

Reply:
just breadcrum to remember to PR IIF for this change too

Reply:
Yes, decodePackedType can be deleted from IIF if we do it this way.